openfisca_core/periods.py

```diff
@@ -711,7 +717,7 @@ def instant(instant):
         return None
     if isinstance(instant, Instant):
         return instant
-    if isinstance(instant, basestring):
+    if isinstance(instant, (str, text_unicode)):
```

Comment:
Excellent question :) 
I think it is because I did a few `(str, text_unicode)` before creating `conv.text_basestring` and did not replace them all :sweat_smile: